gradlew

```diff
@@ -160,6 +160,9 @@ save () {
     echo " "
 }
 APP_ARGS=$(save "$@")
+# Realm's work-around for a bug in Gradle 4.1
```

Comment:
Added a link to bug report.

Currently, we are using `GradleBuild` task only in root project.
So modifying just `gradlew` in root directory is enough.

Do you prefer fixing all `gradlew` files?